docs/developers/tutorials/crowdfunding-p1.md

```diff
@@ -72,13 +72,13 @@ The source code of each smart contract requires its own folder. You'll need to c
 ```bash
 mkdir -p ~/MultiversX/SmartContracts
 cd ~/MultiversX/SmartContracts
-mxpy contract new crowdfunding --template adder
+mxpy contract new crowdfunding --template empty
```

Comment:
We have a brand new templating tool, released yesterday, going to be improved with the next release today.

Might be a good time to talk about that in the docs.

Reply:
When the tool is ready, it will be integrated into `mxpy contract new` :+1: